backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -14,9 +14,16 @@ You back up applications by creating a `Backup` custom resource (CR). See xref:.
 
 For more information about CSI volume snapshots, see xref:../../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots[CSI volume snapshots].
 
-:FeatureName: The `CloudStorage` API for S3 storage
+:FeatureName: The `CloudStorage` API, which automates the creation of a bucket for object storage,
 include::snippets/technology-preview.adoc[]
 
+[NOTE]
+====
+The `CloudStorage` API is a Technology Preview when you use a `CloudStorage` object and want OADP to use the `CloudStorage` API to automatically create an S3 bucket for use as a `BackupStorageLocation`.
```

Comment:
```suggestion
The `CloudStorage` API is a Technology Preview feature when you use a `CloudStorage` object and want OADP to use the `CloudStorage` API to automatically create an S3 bucket for use as a `BackupStorageLocation`.
```

Reply:
@JoeAldinger 
I don't have experience on how to include limitations in the Tech preview admonition and I'm not sure what is acceptable according to the OCP guidelines.
How would you suggest incorporating the 3rd NOTE in the Tech preview admonition?

Reply:
@CarmiWisemon I can't speak for @JoeAldinger, but would you feel comfortable removing the first of the three stacked admonitions? The same note appears in two other places in the assembly. That would solve the triple-stacked notes.

Reply:
I agree with @mburke5678